src/components/commons/SectionGridLayout.tsx

```diff
@@ -32,8 +39,9 @@ export function SectionGridLayout(props: SectionGridLayoutProps): JSX.Element {
 
   return (
     <section
-      className="relative pb-24 pt-0 md:pb-28 lg:pb-[120px] lg:mb-[120px]"
-      data-testid="whyDEXsection"
+      className="relative pb-24 pt-0 md:pb-28 lg:pb-[120px] lg:mb-[120px] scroll-mt-40"
+      id={displayId}
```

Comment:
what's the use case of id?

Reply:
for the `start exploring` button to work